packages/site/src/layout/ComponentView.tsx

```diff
@@ -118,6 +126,10 @@ export const ComponentView = () => {
         return false;
       }
 
+      if (!PageMeta?.notes && index === 3) {
```

Comment:
This also looks okay. We'll have to adjust the auto-links after we merge, but not a huge lift to do that.